crates/edr_eth/src/signature.rs

```diff
@@ -98,23 +47,10 @@ pub enum SignatureError {
     RecoveryError,
 }
 
-/// Recovery message data.
-///
-/// The message data can either be a binary message that is first hashed
-/// according to EIP-191 and then recovered based on the signature or a
-/// precomputed hash.
-#[derive(Clone, Debug, PartialEq, Eq)]
-pub enum RecoveryMessage {
-    /// Message bytes
-    Data(Vec<u8>),
-    /// Message hash
-    Hash(B256),
-}
-
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 /// An ECDSA signature
-pub struct Signature {
+pub struct Ecdsa {
```

Comment:
The name ECDSA is a bit weird for me for a signature struct, as ECDSA is the algorithm and the signature is its output, but I see that it's used as `signature::Ecdsa` so that makes it unambiguous, so not a change request, just feedback.